.kuzzlerc.sample

```diff
@@ -476,19 +476,29 @@
     // (see also https://github.com/luin/ioredis)
     //
     // 1. using a single Redis database:
-    //   node:
+    //   * node:
     //     * host:
     //         The host on which Redis can be reached.
     //         Can take an IP address, an URI or a hostname
     //     * port:
     //         The port on which Redis is running its database:
-    //     * (optional) database:
+    //     * (optional) database (DEPRECATED):
     //         ID of the redis database (default: 0)
+    //         NOTE: this option is deprecated and will be removed in Kuzzle v3.
```

Comment:
Are we sure we want to say it will be removed in v3?
We just pushed it from v2 to v3... :lollipop:

Reply:
I just copy deprecation message for pcre support https://github.com/kuzzleio/kuzzle/blob/8668a12df05315686520d9043d1fdd5b784d22de/.kuzzlerc.sample#L739

Reply:
Yep, I figured... Just saying that how are we sure we will remove it v3? :headphones:

Reply:
Because we always remove deprecated feature when release a new major version 😉

Reply:
Nope. Some deprecated features might stay in the next major version, for instance if the deprecation occurs not too long before the new major release, to allow our users to adjust.